updating deploy setup (#20)
infinum/eightshift-deploy-actions-public: README.md

### Other secrets

_Responsible person: DevOps or TL + Developer_

- `SLACK_WEBHOOK` - webhook for the Slack notifications (ask your TL for this secret).
- `BUGSNAG_API_KEY` - API key for the Bugsnag notifications (ask your TL for this secret).

### Adding custom secrets in the deploy process

_Responsible person: DevOps or TL + Developer_

> To add custom secrets in the deploy process add a custom step after the `Setup WordPress` step in the deploy action. Keep in mind you that the value is always wrapper in single quotes.

Reviewer: ```suggestion
To add custom secrets in the deploy process add a custom step after the `Setup WordPress` step in the deploy action. Keep in mind you that the value is always wrapped in single quotes.
```

Typo?

> To add custom secrets in the deploy process add a custom step after the `Setup WordPress` step in the deploy action. Keep in mind you that the value is always wrapper in single quotes.

Reviewer 2: ```suggestion
To add custom secrets in the deploy process add a custom step after the `Setup WordPress` step in the deploy action. Keep in mind that the value is always wrapped in single quotes.
```

> To add custom secrets in the deploy process add a custom step after the `Setup WordPress` step in the deploy action. Keep in mind you that the value is always wrapped in single quotes.

Reviewer 2: ```suggestion
To add custom secrets in the deploy process add a custom step after the `Setup WordPress` step in the deploy action. Keep in mind that the value is always wrapped in single quotes.
```

```yaml
	- name: Setup custom secrets as environment variables
		shell: bash
		run: |
			php wp-cli.phar config set S3_UPLOADS_BUCKET_URL '${{ secrets.S3_UPLOADS_BUCKET_URL }}'
			php wp-cli.phar config set S3_UPLOADS_BUCKET '${{ secrets.S3_UPLOADS_BUCKET }}'

			# if you need boolean value use --raw flag
			php wp-cli.phar config set QM_DB_SYMLINK false --raw
			...
```

## Env secrets/variables vs Repository secrets/variables

GitHub offers two types of secrets: variables and secrets. The difference between them is that variables are visible in the logs and in the GUI, while secrets are not. Once set secret can't be viewed again.

Also GitHub offers two types of usage for secrets: repository secrets and environment secrets. The difference between them is that repository secrets are available to all workflows in the repository, while environment secrets are available only to the workflows in the environment and this environment is defined in the workflow file.

> When setting up secrets, make sure to use environment secrets make sure you add them to the correct location and correct type based on the sensitivity of the data.

Reviewer: This is a bit hard to read. Possible typo or missing comma?

> When setting up secrets, make sure to use environment secrets make sure you add them to the correct location and correct type based on the sensitivity of the data.

Reviewer 2: ```suggestion
When setting up secrets, make sure to use environment secrets and make sure you add them to the correct location and correct type based on the sensitivity of the data.
```
